recipes/pthread-stubs/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+./configure --prefix="$PREFIX"
+make
+make install
```

Comment:
@ccordoba12 I got lost in the recipes I reviewed. Is this Good to go? Did we talk about test in this one?